### PR TITLE
Manage Applicants / Updates to Campaigns

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: app/controllers/applicant_results_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class ApplicantResultsController < ApplicationController
+  before_action :authenticate_administrator!
+  before_action :set_applicant
+
+  private
+
+  def set_applicant
+    @applicant = Applicant.find(params[:applicant_id])
+  end
+end
```
Comment (hopsoft) [inline app/controllers/applicant_results_controller.rb]: Can this be handled by `ApplicantsController`? If not I wonder if we should treat this like the other search controllers?

File: app/models/applicant.rb [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    #   *First Name:* #{advertiser_params[:first_name]}
+    #   *Last Name:*  #{advertiser_params[:last_name]}
+    #   *Email:*  #{advertiser_params[:email]}
+    #   *Company:*  #{advertiser_params[:company_name]}
+    #   *Monthly Budget:*  #{advertiser_params[:monthly_budget]}
+    #   *Website:*  #{advertiser_params[:company_url]}
+    # MESSAGE
+    # ApplicantMailer.with(form: advertiser_params.to_h).advertiser_application_email.deliver_later
+    # redirect_to advertisers_path, notice: "Your request was sent successfully. We will be in touch."
+  end
```
Comment (hopsoft) [inline app/models/applicant.rb]: Let's name the stubs a little better.

File: app/controllers/applicant_results_controller.rb (hunk 1 above)
Comment (coderberry) [reply app/controllers/applicant_results_controller.rb]: This isn't a search controller, it's a tabbed controller (like `campaign_budgets_controller.rb`)

File: app/models/applicant.rb (hunk 2 above)
Comment (coderberry) [reply app/models/applicant.rb]: Doh!

**Pull request merged**